src/main/java/util/HttpRequestUtils.java

```diff
@@ -1,109 +1,133 @@
 package util;
 
+import com.google.common.base.Strings;
+import com.google.common.collect.Maps;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.util.Arrays;
 import java.util.Map;
 import java.util.stream.Collectors;
 
-import com.google.common.base.Strings;
-import com.google.common.collect.Maps;
-
 public class HttpRequestUtils {
-    /**
-     * @param queryString은
-     *            URL에서 ? 이후에 전달되는 field1=value1&field2=value2 형식임
-     * @return
-     */
-    public static Map<String, String> parseQueryString(String queryString) {
-        return parseValues(queryString, "&");
-    }
 
-    /**
-     * @param 쿠키
-     *            값은 name1=value1; name2=value2 형식임
-     * @return
-     */
-    public static Map<String, String> parseCookies(String cookies) {
-        return parseValues(cookies, ";");
+  private static final Logger logger = LoggerFactory.getLogger(HttpRequestUtils.class);
+
+  /**
+   * @param queryString URL에서 ? 이후에 전달되는 field1=value1&field2=value2 형식
+   * @return
+   */
+  public static Map<String, String> parseQueryString(String queryString) {
+    return parseValues(queryString, "&");
+  }
+
+  /**
+   * @param cookies 쿠키는 name1=value1; name2=value2 형식
+   * @return
+   */
+  public static Map<String, String> parseCookies(String cookies) {
+    return parseValues(cookies, ";");
+  }
+
+  private static Map<String, String> parseValues(String values, String separator) {
+    if (Strings.isNullOrEmpty(values)) {
+      return Maps.newHashMap();
     }
 
-    private static Map<String, String> parseValues(String values, String separator) {
-        if (Strings.isNullOrEmpty(values)) {
-            return Maps.newHashMap();
-        }
+    String[] tokens = values.split(separator);
+    return Arrays.stream(tokens)
+        .map(HttpRequestUtils::getKeyValueByEqual)
+        .filter(Pair::isNotNull)
```

Comment:
```suggestion
        .filter(Objects::nonNull)
```
이걸 쓰면 추가구현하신걸 제외하실 수 있을거같은데 함 살펴보시겠어요?

Reply:
https://github.com/kakao-2022-summer/be-w234-was/pull/55/commits/48392dd48ccfd0857720ac68b6bfa57874abef39

반영했습니다